pkg/deploy/controller/deployment/factory.go

```diff
@@ -122,6 +126,15 @@ func (factory *DeploymentControllerFactory) Create() controller.RunnableControll
 	}
 }
 
+// setPodAttributes updates the deployer pod attributes based on the deployment strategy configuration
+func (factory *DeploymentControllerFactory) setPodAttributes(strategy *deployapi.DeploymentStrategy, pod *kapi.Pod) {
+	switch strategy.Type {
+	case deployapi.DeploymentStrategyTypeCustom:
+		util.MergeInto(pod.Labels, strategy.Labels, 0)
```

Comment:
@ironcladlou Updated diff applies labels to all deployers and adds a comment. Can you please ask the test bot to take another go at it ?